lms/djangoapps/teams/templates/teams/teams.html

```diff
@@ -11,14 +11,18 @@
 
 <%include file="/courseware/course_navigation.html" args="active_page='teams'" />
 
-<div class="team-tab-content"></div>
+<div class="container">
+    <div class="course-wrapper">
+        <section class="course-content" id="course-content">
```

Comment:
Cool. @frrrances - would you mind cluing me into why this class name was chosen?